packages/apps/helpdesk/src/index.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Module} from '@axelor/aos-mobile-core';
+
+export const HelpDeskModule: Module = {
+  name: 'app-helpdesk',
+  title: 'Helpdesk_Helpdesk',
+  subtitle: 'Helpdesk_Helpdesk',
+  icon: 'life-ring',
```

Comment:
It's nice to create translations but if you do not add them in the module export, they won't work...